src/odemis/driver/andorshrk.py

```diff
@@ -347,12 +350,20 @@ def __init__(self, name, role, device, camera=None, accessory=None,
         dependencies = dependencies or {}
         rng = rng or {}
 
+        self._slit_names = SLIT_NAMES.copy()
+        self._force_slit_max = set()
         slits = slits or {}
-        for i in slits:
+        for i, slitn in slits.items():
```

Comment:
Does there needs to be an underscore (or capital) between slit and n? so slit_n to make it distinct that n refers to name?